### ISSUE TITLE
Neural Style Transfer tutorial has drastically different results than the original paper

### ISSUE THREAD

**Issue by Nick-Morgan** [pytorch/pytorch#43457]
Following the [official tutorial](https://pytorch.org/tutorials/advanced/neural_style_tutorial.html) on van Gogh's Starry Night produces the results below. I've included the results from the [original paper](https://arxiv.org/abs/1508.06576) for comparison. The notebook that generated these plots can be viewed [here](https://colab.research.google.com/drive/1W-oboiUmziK7N6CBKp1y6uQISgukfiZP?usp=sharing). 

![image](https://user-images.githubusercontent.com/23083692/90962458-c7a4e300-e475-11ea-9fc0-4b590a50b8ae.png)

The original paper used Caffe's pre-trained VGG19 network, which has a few differences:

1. The pixels are in the range of `[0, 255]` (Pytorch's was trained in the range of `[0, 1]`)
2. The color channels are in the order `BGR` (Pytorch's is in the order of `RGB`)
3. The method of normalization is different.

Despite these differences, I would expect to see results that are closer to the original paper. What else could be causing such a large difference in results?

**Comment by ngimel** [comment]
discuss.pytorch.org is better suited for questions like this.
